docs/source/tasks.rst

```diff
@@ -1,13 +1,37 @@
+.. _tasks:
+
 Tasks
```

Comment:
Why not create folder with tasks, like you did for models i.e create tasks.rst, and folder tasks where each task would be describe?